setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = ansible
+description = Radically simple IT automation
+author_email = info@ansible.com
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
```

Comment:
I've seen that tox.ini lacks py37, I assume that as this is a WIP you'll consider adding it there too